### PR TITLE
Onboard Boilerplate

### PR DESCRIPTION
**Pull request by dustman9000**
Changes necessary to bring certman-operator onboarded with the boilerplate framework.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,19 +1,11 @@
...
+.PHONY: boilerplate-update
+boilerplate-update:
+	@boilerplate/update
 
-# Push the docker image
-.PHONY: docker-push
-docker-push: push
+# TODO: Remove below targets once prow config standardized
+.PHONY: gobuild
+gobuild: go-build
```
Comment (2uasimojo) [inline Makefile:11]: Don't need to do this, since prow isn't actually using `gobuild` directly. Instead, fix the [callout in the Dockerfile](https://github.com/openshift/certman-operator/pull/170/files?file-filters%5B%5D=.go&file-filters%5B%5D=.mk&file-filters%5B%5D=No+extension#diff-5aecd73bbcb9b7410938673269f0d4e676dd18509277b0ff2f2787c9d3e22604R8).

**Review by 2uasimojo — COMMENTED** [review]
This is legit, thanks @dustman9000. Just one thing that can be cleaned up.

**Comment by 2uasimojo** [comment]
Oh, one more thing: `hack/codecov.sh` can be removed. (`app_sre_pr_check.sh` and `validate_yaml.py` can also go away, but only after the app-interface ci.ext job is gone. So nbd if you want to clean up all three at the same time.)

File: Makefile (hunk 1 above)
Comment (dustman9000) [reply Makefile:11]: Thanks, I've updated this.

File: cmd/manager/main.go [hunk 2]
```diff
@@ -34,10 +33,8 @@ import (
 
 // Change below variables to serve metrics on different host or port.
 var (
-	metricsHost                   = "0.0.0.0"
 	metricsPath                   = "/metrics"
 	metricsPort                   = "8080"
-	secretWatcherScanInterval     = time.Duration(10) * time.Minute
```
Comment (jharrington22) [inline cmd/manager/main.go:40]: @dustman9000 are these changes intended?

File: pkg/clients/aws/route53.go [hunk 3]
```diff
@@ -52,63 +52,6 @@ type awsClient struct {
 	client route53iface.Route53API
 }
 
-// searchForHostedZone finds a hostedzone when given an aws client and a domain string
```
Comment (jharrington22) [inline pkg/clients/aws/route53.go:55]: @dustman9000 these too? There are a bunch of other edits after this that I wont comment on

**Review by jharrington22 — CHANGES_REQUESTED** [review]

File: cmd/manager/main.go (hunk 2 above)
Comment (jharrington22) [reply cmd/manager/main.go:40]: Ah just saw the commit message, I figured they were from linting.

**Review by jharrington22 — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (2uasimojo) [reply Makefile]: :+1:

**Comment by 2uasimojo** [comment]
This is good to go once freeze is lifted.

/lgtm

**Comment by 2uasimojo** [comment]
/retest

Merge blocker closed.

**Review by yithian — APPROVED** [review]
